### DISCUSSION TITLE
How to pass environment variables to wine program itself?  "LANG=ja_JP.UTF-8 wine game.exe" for example

### DISCUSSION THREAD

**Discussion by whp199** [bottlesdevs/Bottles#698] [Q&A] [answered]
I see "run with arguments" but that just passes command line arguments to the exe file being run.  I want to pass arguments to wine itself.  How can I do that?
Starting the flatpak with "LANG=ja_JP.UTF-8 flatpak run com.usebottles.bottles" does not help..
I'd just like to run a game that requires Japanese windows locale..

**Comment by mirkobrombin** [comment, marked as answer]
Are you talking about environment variables? Or wine parameters like `--version`?

**Reply by whp199** [reply]
Yes, I'm referring to changing the environment variables for wine itself, not passing parameters to wine.

**Reply by mirkobrombin** [reply]
Environment variables can be set from bottle preferences 
![image](https://user-images.githubusercontent.com/9479903/140624180-3e16d108-8a8f-4a27-a7e8-e9b5790aade6.png)

For wine arguments instead you can use them in this way:
```bash
-winearg1 -winearg2 %command% -progarg1 -progarg2
```

**Reply by whp199** [reply]
Thanks!!
